### DISCUSSION TITLE
Model under-represents night bird

### DISCUSSION THREAD

**Discussion by alexbelgium** [Nachtzuster/BirdNET-Pi#51] [Q&A] [answered]
Night birds such as owls have a low probability threshold in the database even for highly common birds
I guess birdwatchers spend more time observing during the day than the night

Couldn't we correct that manually by correcting the bias ?

**Comment by alexbelgium** [comment]
It would be great to be able to tweak the "Species Occurence Frequency Threshold" for some species that we know are specific of our region

**Comment by MueJosh** [comment]
> It would be great to be able to tweak the "Species Occurence Frequency Threshold" for some species that we know are specific of our region

That's always been possible 

Tools -> Settings -> Model 
![image](https://github.com/Nachtzuster/BirdNET-Pi/assets/145150281/07260787-adeb-4295-bd96-16673df43b50)

**Comment by alexbelgium** [comment]
Thanks! What I meant is by specie : for example I need to use 0.008 to see the Strix Aluco (species occurrence of 0.0085) despite it being a very common night bird in my region, which means I see detections of very improbable birds such as Jynx Torquilla. Also, the Athene noctua also has globally the same value despite being quite rare in my region.

To cope for this ice set a very low global threshold and have a quite significant list of excluded birds.

What would be great would be to say that Strix Aluco for me has an occurrence of 0.5 instead, as I know that a song at night is very likely it!

**Comment by MueJosh** [comment]
> To cope for this ice set a very low global threshold and have a quite significant list of excluded birds.
> 
> 
> 
> What would be great would be to say that Strix Aluco for me has an occurrence of 0.5 instead, as I know that a song at night is very likely it!

Mhmm, I see.. 
I unfortunately I can't really help you with that.


I'm also using BirdNET-GO, it has an feature that you maybe like. 
You can set the detection probability for all species by yourself. 

I have it set up with the lowest species occurrence so it detects ALL birds but the detection probability is set to 0.80 (80%) 
Owls and the birds of interest are set to 0.2 or whatever probability is preferred.

It's not that nice too look at, but it works great! (Still under construction)
I have it running on three RP 02W in a docker container and it's drawing much less current then BirdNET-Pi (on a RP 02W)

**Reply by alexbelgium** [reply]
thanks for the info I'll look at it !

**Comment by Nachtzuster** [comment]
@alexbelgium You could try with the Species Range V2.4-**v1**: uncheck the box in @MueJosh screenshot
V2 is much stricter it seems. (I kinda regret making it the default)

**Reply by alexbelgium** [reply]
Thanks! When using v1 is the threshold still used? As whether I check it or not it seems to me that the "Species list tester" returns the same output?

I've also posted on the main birdnet repo https://github.com/kahst/BirdNET-Analyzer/issues/296 as someone has a similar observation of suspected bias in the model

**Reply by alexbelgium** [reply]
Actually fixed by https://github.com/Nachtzuster/BirdNET-Pi/pull/110

**Comment by gilpayson** [comment]
When I lower the Species occurence threshold, all kinds of weirdness starts showing up.

Question:  Are we able to edit the model itself?

**Reply by alexbelgium** [reply]
We can't adapt the model - but you can try using species occurrence model 1 or 2.4 ; it seems for some locations one can be better than the other. Other than that, it's playing with exclusions and whitelist. But most importantly, the better your microphone (low Signal-Noise-Ratio) the best your quality of detections

**Reply by alexbelgium** [reply]
Hi, there is now a new feature to allow specific birds to be whitelisted, it should allow this by whitelisting all local birds of prey (most under-represented birds). Those birds will therefore be detected even if they are below the Species Occurence Frequency Threhsold, in addition to all the others

**Comment by gilpayson** [comment]
But when this feature is used, ONLY those birds will be detected - is this
still true?

**Reply by alexbelgium** [reply, marked as answer]
The whitelist actually adds species to the normally accepted ones. I've edited my message above to better reflect that

In summary there is 2 modes :
- "Include mode" : if at least one species is on the "include_species_list.txt", only the species from this list are detected. All others are ignored
- "Normal mode" : all species are detected that fulfill the following criteria
    - NOT in the exclude list (exclude_species_list.txt)
    - WITH an occurence above the species occurence frequency threshold defined OR on the whitelist (whitelist_species_list.txt)
